### PR TITLE
Removed phoron.

### PR DESCRIPTION
**Pull request by MistakeNot4892**
- Various constants/vars/behaviors that were named or hardcoded to phoron have been generalized to use stuff like `fuel_value`.
- Phoron in chemistry is replaced by crystalizing agent or enzyme.
- Phoron in gas sim is replaced usually by hydrogen and oxygen.
- Supermatter emits supermatter gas instead of phoron gas. 
- Phoron pistol is plasma pistol.

Opening primarily for Travis, still need to update supermatter to work as its own gas and do more thorough testing.

### CODE REVIEW

File: code/_onclick/hud/screen_objects.dm [hunk 1]
```diff
@@ -260,7 +260,7 @@
 									if (!isnull(t.manipulated_by) && t.manipulated_by != C.real_name && findtext(t.desc,breathes))
 										contents.Add(t.air_contents.total_moles)	//Someone messed with the tank and put unknown gasses
 										continue					//in it, so we're going to believe the tank is what it says it is
-									if(t.air_contents.gas[breathes] && !t.air_contents.gas[MAT_PHORON])
+									if(t.air_contents.gas[breathes] && length(t.air_contents.gas) == 1)
```
Comment (comma) [inline code/_onclick/hud/screen_objects.dm]: this going to cause internals button pick CO2 jetpacks / whatever really over air tanks (that are not pure oxygen) (yes those still exist)
maybe check that none of species set toxic gases are in there

File: code/modules/materials/material_sheets.dm [hunk 2]
```diff
@@ -256,6 +242,20 @@
 /obj/item/stack/material/mhydrogen/ten
 	amount = 10
 
+//Fuel for PACMAN generator.
+/obj/item/stack/material/graphite
+	name = "tritium"
```
Comment (comma) [inline code/modules/materials/material_sheets.dm]: >graphite
>tritium

Comment (MistakeNot4892) [reply code/modules/materials/material_sheets.dm]: Copypaste strikes again.

File: code/modules/materials/definitions/materials_mineral.dm [hunk 3]
```diff
@@ -219,69 +219,24 @@
...
-	ore_icon_overlay = "gems"
-	sale_price = 5
-	value = 200
-	//Note that this has a significant impact on TTV yield.
-	//Because it is so high, any leftover phoron soaks up a lot of heat and drops the yield pressure.
-	gas_specific_heat = 200	// J/(mol*K)
-	//Hypothetical group 14 (same as carbon), period 8 element.
-	//Using multiplicity rule, it's atomic number is 162
-	//and following a N/Z ratio of 1.5, the molar mass of a monatomic gas is:
-	gas_molar_mass = 0.405	// kg/mol
```
Comment (comma) [inline code/modules/materials/definitions/materials_mineral.dm:255]: probably should leave the gas stuff

File: code/modules/modular_computers/file_system/programs/engineering/supermatter_monitor.dm [hunk 4]
```diff
@@ -108,14 +108,12 @@
 			data["SM_gas_O2"] = round(100*air.gas[MAT_OXYGEN]/air.total_moles,0.01)
 			data["SM_gas_CO2"] = round(100*air.gas[MAT_CO2]/air.total_moles,0.01)
 			data["SM_gas_N2"] = round(100*air.gas[MAT_NITROGEN]/air.total_moles,0.01)
-			data["SM_gas_PH"] = round(100*air.gas[MAT_PHORON]/air.total_moles,0.01)
```
Comment (comma) [inline code/modules/modular_computers/file_system/programs/engineering/supermatter_monitor.dm:111]: also if SM is intended to output SM gas should probably track it here

File: code/modules/supermatter/setup_supermatter.dm [hunk 5]
```diff
@@ -41,9 +40,6 @@
 			if("CO2")
 				C.canister_type = /obj/machinery/portable_atmospherics/canister/carbon_dioxide/engine_setup/
 				continue
-			if("PH")
```
Comment (comma) [inline code/modules/supermatter/setup_supermatter.dm:44]: here too I thought SM gas was supposed to replace it

File: nano/templates/supermatter_monitor.tmpl [hunk 6]
```diff
@@ -83,12 +83,6 @@
 					<div class="itemContent">
 						{{:data.SM_gas_N2}} %
 					</div>
-					<div class="itemLabel">
-						PH:
```
Comment (comma) [inline nano/templates/supermatter_monitor.tmpl:87]: SM gaaaas

**Comment by MistakeNot4892** [comment]
Need to redo this from scratch at this point.

**Pull request closed without merging**